Source/WebKit/UIProcess/Network/NetworkProcessProxy.h

```diff
@@ -450,6 +451,7 @@ class NetworkProcessProxy final : public AuxiliaryProcessProxy, private ProcessT
 #endif
 
     WeakHashSet<WebsiteDataStore> m_websiteDataStores;
+    HashMap<PAL::SessionID, CompletionHandler<void(String)>> m_removalCompletionHandlers;
```

Comment:
This name is too generic given its location. What's being removed? I'd suggest something like `m_sessionRemovalCompletionHandlers`.

Reply:
Removed.